core/iwasm/aot/aot_loader.c

```diff
@@ -1575,7 +1575,7 @@ load_object_data_sections(const uint8 **p_buf, const uint8 *buf_end,
         /* Allocate memory for data */
         if (data_sections[i].size > 0
             && !(data_sections[i].data = os_mmap(NULL, data_sections[i].size,
-                                                 map_prot, map_flags))) {
+                                                 map_prot, map_flags, NULL))) {
```

Comment:
Can pass `os_get_invalid_handle()` instead of `NULL` if we declare os_mmap as `os_mmap(..., os_file_handle file)`

Reply:
Fixed